src/main/resources/templates/fragments.html

```diff
@@ -8,5 +8,19 @@
 </head>
 <body>
 
+<nav th:fragment="header"><!-- Navigation header -->
+    <div style="background: url(https://images.unsplash.com/photo-1470569584027-67bf7fa60807?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80)" class="jumbotron bg-cover text-white">
+        <h1 th:text="${title}">Mental Healthcare Access</h1>
+            <ul class="nav">
+                <li class="nav-item"><a class="nav-link" href="#">Home</a></li>
```

Comment:
Would it be worth redirecting "Home" this to "/". Now it is juts a placeholder but just so we don't forget to edit it later when we actually have content in the main page. Looks so good; major improvement on how it looks!